util/grub-switch-to-blscfg.in

```diff
@@ -252,7 +252,7 @@ if [[ "${GENERATE}" -eq 1 ]] ; then
         cp -af "${GRUB_CONFIG_FILE}${backupsuffix}" "${GRUB_CONFIG_FILE}"
```

Comment:
The patch looks good to me, but your subject line and commit message isn't accurate. You should write `${etcdefaultgrub}` instead of `$(etcdefaultgrub}`.

Reply:
It isn't too nice solution, but I have fixed it.